### PR TITLE
Updated the wg06 driver so that it calls addForceTorque to register itse...

### PR DESCRIPTION
**Pull request by isura**
...lf

### CODE REVIEW

File: ethercat_hardware/include/ethercat_hardware/wg06.h [hunk 1]
```diff
@@ -182,6 +182,7 @@ class WG06 : public WG0X
 
   pr2_hardware_interface::PressureSensor pressure_sensors_[2];
   pr2_hardware_interface::Accelerometer accelerometer_;
+  pr2_hardware_interface::ForceTorque ft_sensor_;
```
Comment (ahendrix) [inline ethercat_hardware/include/ethercat_hardware/wg06.h]: This already exists as force_torque_ around line 235.

File: ethercat_hardware/src/wg06.cpp [hunk 2]
```diff
@@ -348,6 +348,15 @@ bool WG06::initializeFT(pr2_hardware_interface::HardwareInterface *hw)
               ft_raw_analog_in_.name_.c_str(), sh_->get_ring_position());
     return false;
   }
+
+  // Register force/torque sensor with pr2_hardware_interface::HardwareInterface
+  ft_sensor_.name_ = actuator_.name_;
+  if (hw && !hw->addForceTorque(&ft_sensor_))
+  {
```
Comment (ahendrix) [inline ethercat_hardware/src/wg06.cpp:355]: This should use the existing force_torque_ member.

File: ethercat_hardware/include/ethercat_hardware/wg06.h (hunk 1 above)
Comment (isura) [inline ethercat_hardware/include/ethercat_hardware/wg06.h]: Removed it.

File: ethercat_hardware/src/wg06.cpp (hunk 2 above)
Comment (isura) [inline ethercat_hardware/src/wg06.cpp:355]: Changed to use existing force_torque_ member.

**Pull request merged**